README.md

```diff
@@ -1 +1,24 @@
 # hw01
+
+# INTRODUCTION
+My name is Nairi Brown. I am from Massachusetts. I also went to UChicago for undergrad. 
+At home, I have a wonderful dog named **Louie** who is adorable. 
+
+*see picture below*
+
+![](lou.jpeg)
```

Comment:
Your dog is very cute! Not to repeat the professor, but perhaps you could have shrunk the image down a little.